src/index.js

```diff
@@ -0,0 +1,17 @@
+import 'lodash';
+import './style.css';
+import listArr from './list.js';
+
+function buildtable(item) {
```

Comment:
- [ ] Please, use an arrow function here, this is more according to good practices with ES6. 🐶